packages/subscription-manager/src/components/TabsSwitcher/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.section-nav {
+	box-shadow: none !important;
+	border-bottom: 32px;
+
+	.section-nav-tabs {
+		display: flex;
+
+		.section-nav-tabs__list {
+			box-shadow: inset 0 -1px 0 rgba($studio-black, 0.05);
+			gap: 16px;
+			flex-basis: fit-content;
+		}
+	}
+
+	.section-nav-tab {
```

Comment:
Since we are already opening up the definition for `.section-nav-tabs`  and `.section-nav-tabs__list`, we may as well put the `.section-nav-tab` within to represent the expected structure within the DOM. 🙂 I will do that, and hopefully you do not mind. If you have objections as to why this is not necessarily a good idea, please bring this up, it will help me to learn. 🙂